src/test/java/com/aws/greengrass/logmanager/LogManagerServiceTest.java

```diff
@@ -587,28 +593,49 @@ void GIVEN_cloud_watch_attempt_handler_WHEN_attempt_completes_THEN_successfully_
 
         CloudWatchAttempt attempt = new CloudWatchAttempt();
         Map<String, CloudWatchAttemptLogInformation> logStreamsToLogInformationMap = new HashMap<>();
-        File file1 = new File(getClass().getResource("testlogs2.log").toURI());
-        File file2 = new File(getClass().getResource("testlogs1.log").toURI());
-        Map<String, CloudWatchAttemptLogFileInformation> attemptLogFileInformationMap1 = new HashMap<>();
-        attemptLogFileInformationMap1.put(file1.getAbsolutePath(), CloudWatchAttemptLogFileInformation.builder()
+
+        LogFile lastProcessedFile = new LogFile(directoryPath.resolve("testlogs2.log").toUri());
```

Comment:
This test can be refactored. See if you can remove the duplication and extract what is similar.